packages/editor-ui/src/components/executions/workflow/WorkflowExecutionAnnotationPanel.vue

```diff
@@ -0,0 +1,385 @@
+					v-bind:key="attr"
+					:class="$style.customDataEntry"
+				>
+					<n8n-text :class="$style.key" size="small" color="text-base">
+						{{ attr }}
+					</n8n-text>
+					<n8n-text :class="$style.value" size="small" color="text-base">
+						{{ activeExecution?.customData[attr] }}
+					</n8n-text>
+				</div>
+			</div>
+			<div v-else :class="$style.noResultsContainer" data-test-id="execution-list-empty">
+				<n8n-text color="text-base" size="small" align="center">
+					<span v-html="$locale.baseText('executionAnnotationView.data.notFound')" />
+				</n8n-text>
+			</div>
+		</div>
+	</div>
+</template>
+
+<style module lang="scss">
+.container {
+	z-index: 1;
+	position: absolute;
+	bottom: 0;
+	right: var(--spacing-xl);
+	transform: translate(0, 100%);
+	max-height: calc(100vh - 250px);
+	width: 250px;
+
+	display: flex;
+	flex-direction: column;
+	overflow: auto;
+
+	background-color: var(--color-background-xlight);
+	border: var(--border-base);
+	border-radius: var(--border-radius-base);
+}
+
+.section {
+	padding: var(--spacing-s);
+	display: flex;
+	flex-direction: column;
+
+	&:not(:last-child) {
+		border-bottom: var(--border-base);
+	}
+}
+
+.metadata {
+	padding-top: var(--spacing-s);
+}
+
+.heading {
+	display: flex;
+	justify-content: space-between;
+	align-items: baseline;
+	padding-right: var(--spacing-l);
+}
+
+.controls {
+	padding: var(--spacing-s) 0 var(--spacing-xs);
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding-right: var(--spacing-m);
+
+	button {
+		display: flex;
+		align-items: center;
+	}
+}
+
+.vote {
+	padding: 0 0 var(--spacing-xs);
+	font-size: var(--font-size-xs);
+	flex: 1;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+
+	.ratingIcon {
+		display: flex;
+		flex-direction: row;
+
+		.highlight {
+			color: var(--color-primary);
+		}
+	}
+}
+
+.customDataEntry {
+	display: flex;
+	flex-direction: column;
+
+	&:not(:first-of-type) {
+		margin-top: var(--spacing-s);
+	}
+
+	.key {
+		font-weight: bold;
+	}
+}
+
+.executionList {
+	flex: 1;
+	overflow: auto;
+	margin-bottom: var(--spacing-m);
+	background-color: var(--color-background-xlight) !important;
+
+	// Scrolling fader
+	&::before {
+		position: absolute;
+		display: block;
+		width: 270px;
+		height: 6px;
+		background: linear-gradient(to bottom, rgba(251, 251, 251, 1) 0%, rgba(251, 251, 251, 0) 100%);
+		z-index: 999;
+	}
+
+	// Lower first execution card so fader is not visible when not scrolled
+	& > div:first-child {
+		margin-top: 3px;
+	}
+}
+
+.infoAccordion {
+	position: absolute;
+	bottom: 0;
+	margin-left: calc(-1 * var(--spacing-l));
+	border-top: var(--border-base);
+
+	& > div {
+		width: 309px;
+		background-color: var(--color-background-light);
+		margin-top: 0 !important;
+	}
+}
+
+.noResultsContainer {
+	width: 100%;
+	margin-top: var(--spacing-s);
+	//text-align: center;
+}
+</style>
+
+<style lang="scss" scoped>
```

Comment:
Do we need to mix scoped and module styles? Why couldn't we just use module?

Reply:
I moved all styles to module, but had to add `tags-container` class onto the element, because it is defined globally in other files. Would be nice to refactor, but out of the scope of this PR.